analyzers/src/SonarAnalyzer.CFG/LiveVariableAnalysis/RoslynLiveVariableAnalysis.cs

```diff
@@ -161,6 +165,17 @@ private void BuildBranchesFinally(BasicBlock source, ControlFlowRegion finallyRe
         }
     }
 
+    private void BuildBranchesNestedCatchRethrow(BasicBlock block)
+    {
+        var tryRegion = block.EnclosingRegion(ControlFlowRegionKind.TryAndCatch).NestedRegion(ControlFlowRegionKind.Try);
+        foreach (var catchBlock in tryRegion.ReachableHandlers()
+                                            .Where(x => x.Kind == ControlFlowRegionKind.Catch && x != block.EnclosingRegion && x.FirstBlockOrdinal > block.Ordinal)
+                                            .SelectMany(x => x.Blocks(Cfg)))
```

Comment:
I would separate this into an extra variable. `foreach` with multiple lines is always awkward to read.